package.json

```diff
@@ -28,6 +31,8 @@
     "extends": "stylelint-config-gds/scss"
   },
   "devDependencies": {
+    "jasmine-browser-runner": "^1.0.0",
+    "jasmine-core": "^4.0.1",
```

Comment:
We should add these two to the NPM dependabot configuration: https://github.com/alphagov/service-manual-publisher/blob/849d2a77ecae7ea2357085c10b218a5d00e91828/.github/dependabot.yml#L43-L47

Reply:
👍